src/amp-story-player/amp-story-player-iframe-pool.js

```diff
@@ -14,6 +14,16 @@
  * limitations under the License.
  */
 
+/** @enum {number} */
+const IframePosition = {
+  PREVIOUS: -1,
+  CURRENT: 0,
+  NEXT: 1,
+};
+
+/** @const {number} */
+const MAX_IFRAMES = 2;
```

Comment:
You lost me here, why does the player have two `MAX_IFRAMES` constant that are different?